docs/Guidelines/Iconography/Icons.tsx

```diff
@@ -0,0 +1,299 @@
+import React, { ReactElement, useState } from 'react'
+import styled from '@emotion/styled'
+import * as Icon from '../../../src/icons'
+import { device, fontSize, radius, SmallText, space } from '../../../src'
+
+const IconGrid = styled.div`
+  display: grid;
+  grid-gap: ${space[8]};
+  grid-template-columns: repeat(auto-fit, minmax(${space[128]}, 1fr));
+`
+
+const IconName = styled(SmallText)`
+  font-size: ${fontSize[12]};
+  margin-top: ${space[8]};
+
+  @media ${device.tablet} {
+    font-size: ${fontSize[12]};
+  }
+`
+
+const IconBlock = styled.div`
+  position: relative;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  overflow: hidden;
+  word-wrap: no-wrap;
+  text-align: center;
+  padding: ${space[12]} ${space[4]};
+  background-color: rgb(245, 245, 246); //Stardust
+  border-radius: ${radius.md};
+`
+
+const CopyButton = styled.button`
+  position: absolute;
+  top: ${space[4]};
+  right: ${space[4]};
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+`
+
+interface IconWrapperProps {
+  icon: ReactElement
+  name: string
+}
+
+const IconWrapper: React.FC<IconWrapperProps> = ({ icon, name }) => {
+  const [copied, setCopied] = useState(false)
+
+  return (
+    <IconBlock>
+      {icon}
+      <IconName>{name}</IconName>
+      <CopyButton
+        onClick={() => {
+          setCopied(true)
+          navigator.clipboard.writeText(`<${name} title="${name}" />`)
+
+          setTimeout(() => {
+            setCopied(false)
+          }, 2000)
+        }}
+      >
+        {copied ? (
+          <Icon.Checkmark size={12} />
+        ) : (
+          <Icon.Copy title="Copy name" size={12} />
+        )}
+      </CopyButton>
+    </IconBlock>
+  )
+}
+
+export const Icons = () => {
+  return (
+    <IconGrid>
```

Comment:
Can't we make this a bit smarter by mapping over certain values so we don't have to repeat ourselves? 😛  Maybe you can use something like `Object.keys(Icon)` to map over all the icons we import?

Reply:
No I couldn't find a way to do that with the components 😓

Reply:
😢